src/billing/Project.js

```diff
@@ -671,7 +498,7 @@ const ProjectDetail = ({
               margin: '1rem 1rem 0',
               padding: '1rem',
               border: `1px solid ${colors.warning()}`,
-              backgroundColor: colors.warning(0.15),
+              backgroundColor: colors.warning(0.1), // needs to be sufficient contrast with link color
```

Comment:
Fixes color contrast issue for this warning message:

<img width="1410" alt="image" src="https://github.com/user-attachments/assets/7ffba8af-d874-4873-8245-221fb8ba0d13">